packages/flutter_form_builder/lib/src/options/form_builder_filter_chips_option.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+
+/// An option for filter chips.
+///
+/// The type `T` is the type of the value the entry represents. All the entries
+/// in a given menu must represent values with consistent types.
+class FormBuilderFilterChipsOption<T> extends FormBuilderFieldOption<T> {
+  final Widget? avatar;
+
+  /// Creates an option for fields with selection options
+  const FormBuilderFilterChipsOption({
+    Key? key,
+    required value,
+    this.avatar,
+    child,
```

Comment:
```suggestion
    super.child,
```
Is it possible to perform it?

Reply:
That's an update in dart on latest release. But so far I could not test it. Btw, this is not a required request, I was just asking for a test. Your PR is good for me. The only thing is the owner's request here: https://github.com/danvick/flutter_form_builder/pull/1031#issuecomment-1152160909

Reply:
Thanks for mentioning, I didn't know that it's possible in newer versions.